frontend/components/Pagination.vue

```diff
@@ -101,149 +102,84 @@
-            2
-          </a>
-          <a
-            href="#"
-            class="
-              bg-white
-              border-gray-300
-              text-gray-500
-              hover:bg-gray-50
-              hidden
-              md:inline-flex
-              relative
-              items-center
-              px-4
-              py-2
-              border
-              text-xs
-              md:text-sm
-              font-medium
-            "
-          >
-            3
-          </a>
-          <span
-            class="
-              relative
-              inline-flex
-              items-center
-              px-4
-              py-2
-              border border-gray-300
-              bg-white
-              text-xs
-              md:text-sm
-              font-medium
-              text-gray-700
-            "
-          >
-            ...
           </span>
-          <a
-            href="#"
-            class="
-              bg-white
-              border-gray-300
-              text-gray-500
-              hover:bg-gray-50
-              hidden
-              md:inline-flex
-              relative
-              items-center
-              px-4
-              py-2
-              border
-              text-xs
-              md:text-sm
-              font-medium
-            "
+          <div
+            v-for="current1 in leftSide"
+            :key="current1"
+            @click="changepage(current1)"
           >
-            8
-          </a>
-          <a
-            href="#"
-            class="
-              bg-white
-              border-gray-300
-              text-gray-500
-              hover:bg-gray-50
-              relative
-              inline-flex
-              items-center
-              px-4
-              py-2
-              border
-              text-xs
-              md:text-sm
-              font-medium
-            "
-          >
-            9
-          </a>
-          <a
-            href="#"
-            class="
-              bg-white
-              border-gray-300
-              text-gray-500
-              hover:bg-gray-50
-              relative
-              inline-flex
-              items-center
-              px-4
-              py-2
-              border
-              text-xs
-              md:text-sm
-              font-medium
-            "
+            <span
+              :class="`z-10
+                bg-indigo-50
+                border-indigo-500
+                text-indigo-600
+                relative
+                inline-flex
+                items-center
+                px-4
+                py-2
+                border
+                text-xs
+                md:text-sm
+                font-medium
+                cursor-pointer
+                ${page === current1 ? 'bg-blue' : 'bg-white'}
+                `"
+            >
+              {{ current1 }}
+            </span>
+          </div>
+          <div>
+            <span
+              class="
```

Comment:
Hi @Asam237 you should just change the color of this buton becauce it is the one that has the current page

Reply:
Okay. It's good now @Zaker237 ?
 
![newwww](https://user-images.githubusercontent.com/34966088/154055231-49861b74-9c7f-4925-9e46-0eef52803bb3.png)